src/program/GameLoop.cpp

```diff
@@ -734,13 +735,15 @@ void GameLoop::processInputs(AllInputs &ai)
                     movie.setLockedInputs(ai);
 
                 /* Save inputs to moviefile */
-                movie.inputs->setInputs(ai, keep_inputs);
+                if (modified)
```

Comment:
We don't want the check for "modified" here. The inputs are always stored in the movie because we are in recording mode.